scripts/install-dev.sh

```diff
@@ -515,15 +515,20 @@ install_editable_webui() {
     sed_inplace "s@debug = true@debug = false@" config.toml
     # The webserver endpoint to use in the session mode.
     sed_inplace "s@#apiEndpoint =@apiEndpoint = "'"'"http://127.0.0.1:${WEBSERVER_PORT}"'"@' config.toml
-    sed_inplace "s@#apiEndpointText =@apiEndpointText = "'"'"${site_name}"'"' config.toml
+    sed_inplace "s@#apiEndpointText =@apiEndpointText = "'"'"${site_name}"'"@' config.toml
     # webServerURL lets the electron app use the web UI contents from the server.
     # The server may be either a `npm run server:d` instance or a `./py -m ai.backend.web.server` instance.
     # In the former case, you may live-edit the webui sources while running them in the electron app.
     sed_inplace "s@webServerURL =@webServerURL = "'"'"http://127.0.0.1:${WEBSERVER_PORT}"'"@' config.toml
     sed_inplace "s@proxyURL =@proxyURL = "'"'"http://127.0.0.1:${WSPROXY_PORT}"'"@' config.toml
-    echo "PROXYLISTENIP=0.0.0.0" >> src/ai/backend/webui/.env
-    echo "PROXYBASEHOST=localhost" >> src/ai/backend/webui/.env
-    echo "PROXYBASEPORT=${WSPROXY_PORT}" >> src/ai/backend/webui/.env
+    env_file="src/ai/backend/webui/.env"
+    env_dir=`dirname $env_file`
+    if [ ! -f $env_file ]; then
+      mkdir -p $env_dir; touch $env_file
+    fi
+    echo "PROXYLISTENIP=0.0.0.0" >> $env_file
+    echo "PROXYBASEHOST=localhost" >> $env_file
+    echo "PROXYBASEPORT=${WSPROXY_PORT}" >> $env_file
```

Comment:
We have already `cd`'ed into `src/ai/backend/webui`. So here `$env_file` must be just `.env`. The original path was my mistake.

Reply:
I fixed it. remove env_dir and checking only env_file